2022-01-15 13:59:28,852 INFO    MainThread:142118 [wandb_setup.py:_flush():71] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2022-01-15 13:59:28,853 INFO    MainThread:142118 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 13:59:28,853 INFO    MainThread:142118 [wandb_init.py:_log_setup():371] Logging user logs to /home/marc/development/python/udacity_course/genre_classification/starter/download/wandb/run-20220115_135928-fa1uktix/logs/debug.log
2022-01-15 13:59:28,853 INFO    MainThread:142118 [wandb_init.py:_log_setup():372] Logging internal logs to /home/marc/development/python/udacity_course/genre_classification/starter/download/wandb/run-20220115_135928-fa1uktix/logs/debug-internal.log
2022-01-15 13:59:28,854 INFO    MainThread:142118 [wandb_init.py:init():404] calling init triggers
2022-01-15 13:59:28,854 INFO    MainThread:142118 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-15 13:59:28,854 INFO    MainThread:142118 [wandb_init.py:init():460] starting backend
2022-01-15 13:59:28,855 INFO    MainThread:142118 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 13:59:28,862 INFO    MainThread:142118 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 13:59:28,864 INFO    MainThread:142118 [backend.py:ensure_launched():221] started backend process with pid: 142147
2022-01-15 13:59:28,864 INFO    MainThread:142118 [wandb_init.py:init():469] backend started and connected
2022-01-15 13:59:28,865 INFO    MainThread:142118 [wandb_init.py:init():533] updated telemetry
2022-01-15 13:59:28,870 INFO    MainThread:142118 [wandb_init.py:init():563] communicating current version
2022-01-15 13:59:29,248 INFO    MainThread:142118 [wandb_init.py:init():568] got version response 
2022-01-15 13:59:29,248 INFO    MainThread:142118 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 13:59:29,445 INFO    MainThread:142118 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 13:59:29,911 INFO    MainThread:142118 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 13:59:29,911 INFO    MainThread:142118 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 13:59:29,911 INFO    MainThread:142118 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 13:59:29,912 INFO    MainThread:142118 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-15 13:59:29,912 INFO    MainThread:142118 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 13:59:35,280 INFO    MainThread:142118 [wandb_run.py:finish():1457] finishing run mdunford/exercise_14/fa1uktix
2022-01-15 13:59:35,280 INFO    MainThread:142118 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-15 13:59:35,281 INFO    MainThread:142118 [wandb_run.py:_restore():1752] restore
2022-01-15 13:59:37,932 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1086559
  total_bytes: 1086559
}

2022-01-15 13:59:38,153 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1086559
  total_bytes: 1086559
}

2022-01-15 13:59:38,467 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1086559
  total_bytes: 1088537
}

2022-01-15 13:59:38,570 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1086559
  total_bytes: 1089243
}

2022-01-15 13:59:38,674 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1086559
  total_bytes: 1089243
}

2022-01-15 13:59:38,776 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}

2022-01-15 13:59:38,878 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}

2022-01-15 13:59:38,982 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}

2022-01-15 13:59:39,085 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}

2022-01-15 13:59:39,188 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}

2022-01-15 13:59:39,888 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}

2022-01-15 13:59:40,196 INFO    MainThread:142118 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  artifact_count: 1
}
pusher_stats {
  uploaded_bytes: 1089243
  total_bytes: 1089243
}
local_info {
}

2022-01-15 13:59:41,249 INFO    MainThread:142118 [wandb_run.py:_append_files():2180] logging synced files
